training/src/anemoi/training/diagnostics/callbacks/plot.py

```diff
@@ -824,6 +828,24 @@ def automatically_determine_group(name: str) -> str:
             legend_patches,
         )
 
+    def argsort_name_variablelevel(self) -> list[int]:
```

Comment:
Would it be worth putting this in to a callback or loss utils? I can see this function being useful for other callbacks in the future

Reply:
That's right. Will do that.